bsip-0039.md

```diff
@@ -35,28 +36,33 @@ makes it unnecessary complicated.
 
 # Rational
 
-By proposing an action, the proposer can be considered as an agreeing
-party, otherwise the proposal wouldn't have been created in the first
-place.
+Giving the proposer an option to automatically approve the proposal
+after creation solves the issue.
 
 If the proposer is not part of the multisig-setup, having him approve
-the proposal automatically does affect the validity of the proposal
+the proposal automatically does not affect the validity of the proposal
 itself.
 
-# Specifications
+It's not a security related change, changing the default behavior is not
```

Comment:
Actually the change to improve security indirectly.

A naive user might bundle a `proposal_create` and a `proposal_update` (i. e. approve) operation into one transaction to achieve the same effect. However, the final proposal ID is not known before the `proposal_create` is beyond the last irreversible block. So the naive user might inadvertently approve the wrong proposal.

Reply:
Thanks. Updated.